config/efr32/lib/pw_rpc/pw_rpc.gni

```diff
@@ -19,6 +19,7 @@ pw_log_BACKEND = "$dir_pw_log_basic"
 pw_assert_BACKEND = "$dir_pw_assert_log"
 pw_sys_io_BACKEND =
     "${chip_root}/examples/platform/efr32/pw_sys_io:pw_sys_io_efr32"
+pw_trace_BACKEND = "$dir_pw_trace_tokenized"
```

Comment:
pw_trace does not seem related to rpc, so tying it to pw_rpc.gni does not seem right.

efr32-brd4161a-lock-rpc build fails after this:

```
2022-04-12 09:25:10 INFO    ERROR at //third_party/connectedhomeip/config/efr32/lib/pw_rpc/pw_rpc.gni:22:20: Build argument has no effect.
2022-04-12 09:25:10 INFO    pw_trace_BACKEND = "$dir_pw_trace_tokenized"
2022-04-12 09:25:10 INFO                       ^------------------------
2022-04-12 09:25:10 INFO    The variable "pw_trace_BACKEND" was set as a build argument
2022-04-12 09:25:10 INFO    but never appeared in a declare_args() block in any buildfile.
```

Reply:
created #17278 to move this setting around.